client/my-sites/plans/ecommerce-trial/wooexpress-plans/style.scss

```diff
@@ -16,6 +16,12 @@
 			max-width: 100%;
 			width: 100%;
 		}
+
+		@media (max-width: 375px) {
+			.segmented-control__text {
+				font-size: 0.75rem !important;
```

Comment:
Can we use `$font-body-extra-small` here instead of a hard-coded font value?

Bonus points if you can make the selector more specific to avoid `!important` IMO :D